src/main/java/net/minecraftforge/gui/capability/impl/GuiProviderBlank.java

```diff
@@ -0,0 +1,36 @@
+package net.minecraftforge.gui.capability.impl;
+
+import io.netty.buffer.ByteBuf;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.inventory.Container;
+import net.minecraft.world.World;
+import net.minecraftforge.gui.capability.IGuiProvider;
+
+import javax.annotation.Nullable;
+
+public class GuiProviderBlank implements IGuiProvider<Object>
+{
+    @Override
+    public void deserialize(ByteBuf buffer) { }
+
+    @Override
+    public void serialize(ByteBuf buffer) { }
+
+    @Override
+    public Object getOwner(EntityPlayer player, World world)
+    {
+        return null;
```

Comment:
Maybe you should make this abstract instead of returning null, since the method is not marked `@Nullable`?